src/sage/numerical/interactive_simplex_method.py

```diff
@@ -2547,9 +2548,17 @@ def revised_dictionary(self, *x_B):
                 x_B[self.b().list().index(bm)] = self.auxiliary_variable()
         return LPRevisedDictionary(self, x_B)
 
-    def run_revised_simplex_method(self):
+    def run_revised_simplex_method(self, pivot_rule = None, *args):
         r"""
         Apply the revised simplex method and return all steps.
+                
+        INPUT:
+        
+        - :string: or :SimplexMethodPivot: Optional- set a pivot rule. 
```

Comment:
Please reformat as shown in https://deploy-preview-37575--sagemath.netlify.app/html/en/developer/coding_basics#template